icons/plug-zap-2.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M10 20c-1.1 1.1-3 1-4.1-.1l-1.8-1.8C3 17 2.9 15.1 4 14l2-2 6 6-2 2z" />
```

Comment:
Maybe just something that I think is a pretty neat trick: if you make diagonal corners have a rounding of 2.41px, the corner will be absolutely pixel perfect:
```suggestion
  <path d="m3.7 17.7 2.6 2.6a2.4 2.4 0 0 0 3.4 0L12 18l-6-6-2.3 2.3a2.41 2.41 0 0 0 0 3.4Z" />
```
![image](https://user-images.githubusercontent.com/17746067/231820202-c1c65edd-0030-4ca7-8baf-7b9fcf477787.png)